controllers/glance_controller.go

```diff
@@ -284,15 +283,6 @@ func (r *GlanceReconciler) reconcileInit(
 ) (ctrl.Result, error) {
 	r.Log.Info(fmt.Sprintf("Reconciling Service '%s' init", instance.Name))
 
-	// Define the PVCs objects required by Glance
-	ctrlResult, err := r.ensurePVC(ctx, helper, instance, serviceLabels)
```

Comment:
moving to `statefulset` in the first place allow us to remove the PV creation logic from the `glance-controller`